yapio.c

```diff
@@ -1293,8 +1293,14 @@ yapio_md_buffer_io(const yapio_test_group_t *ytg, const bool dump)
     /* each rank writes its own md file '{filename}-md-{rank}' */
     char md_file[PATH_MAX];
 
-    int rc = snprintf(md_file, PATH_MAX, "%s-md-%d", yapioTestFileName,
+    /* md files are hidden in folder where test is launched
+     * note: yapioTestFileName starts with './' */
+    char filename_hidden[PATH_MAX] = "./.";
+    strncpy(filename_hidden + 3, yapioTestFileName + 2, PATH_MAX);
```

Comment:
you are still suject to length problem.

Reply:
There is a check at the next line with snprintf(), I think the length error would be caught